raft/src/main/java/org/apache/kafka/raft/internals/BatchMemoryPool.java

```diff
@@ -29,15 +29,15 @@
 public class BatchMemoryPool implements MemoryPool {
```

Comment:
Could we update the javadoc above to match current behavior?

Reply:
Done.